### PR TITLE
data page UI improvements

### PR DESCRIPTION
**Pull request by cmperez**

### CODE REVIEW

File: client/js/components/data_page.jsx [hunk 1]
```diff
@@ -69,12 +86,21 @@ var TableHeadersRow = React.createClass({
       }
     });
   },
+  statics: {
+    getHeaderClass: function(index) {
+      if (index % 2 === 0) {
```
Comment (dduugg) [inline client/js/components/data_page.jsx]: would this be clearer with a ternary?

**Comment by dduugg** [comment]
:+1: I'd prefer ternaries in place of the `if` statements, but that's your call

File: client/js/components/data_page.jsx (hunk 1 above)
Comment (cmperez) [inline client/js/components/data_page.jsx]: absolutely. I'll make the change.

**Comment by cmperez** [comment]
using ternary operators now

File: client/styles/style.css [hunk 2]
```diff
@@ -345,21 +345,88 @@ select {
...
+  color: teal;
+}
+
+.upload {
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
```
Comment (dduugg) [inline client/styles/style.css]: is the use of `px` here and `em` for the other `font-size` properties intentional? is there a method to the order of the properties (`font-size` after `padding` here, but the other way around in `.download-button, .upload-button`)?

File: client/js/components/data_page.jsx [hunk 3]
```diff
@@ -1,6 +1,5 @@
 var React = require("react");
 var _ = require("lodash");
-var reactBootStrap = require("react-bootstrap");
```
Comment (dduugg) [inline client/js/components/data_page.jsx]: can the dependency be removed?

File: client/styles/style.css (hunk 2 above)
Comment (cmperez) [inline client/styles/style.css]: generally I try to keep everything ordered alphabetically and use `em` for `font-size` as a best practice (scales much better on mobile device screen resolutions). you might be able to guess that I hastily copy pasted the `upload-button` and `upload` classes from a blog

File: client/js/components/data_page.jsx (hunk 3 above)
Comment (cmperez) [inline client/js/components/data_page.jsx]: most likely. just need to make sure rachel isn't using it

**Comment by dduugg** [comment]
:star2: :+1: :star2:

**Comment by zindlerb** [comment]
:sparkles: :+1: :sparkles:

**Pull request merged**